frontend/src/types/enum/DetektierteVerkehrsart.ts

```diff
@@ -0,0 +1,6 @@
+export enum DetektierteVerkehrsart {
```

Comment:
In der GUI des Frontend und Adminportal nennt sich der Enum `DetektierteVerkehrsart `.

Im Backend ist die Benamsung `Verkehrsart`.

In der Geodaten-EAI nennt sich der Enum `Verkehrsarten`.

Der Name sollte in den GUIs, im Backend und in der Geodateneai gleichgezogen werden.